examples/ezstream-full.xml

```diff
@@ -110,7 +110,13 @@
     <!-- Program to query for metadata instead of from media files -->
     <!-- <program>metadata.sh</program> -->
 
-    <!-- Metadata format -->
+    <!-- 
```

Comment:
The comments in this example files are meant as concise hints about what a setting refers to, because it's so large, and not (a copy of the) documentation. Please revert this change and update doc/ezstream.1.in.in instead.